bin/workflows/pycbc_make_inference_inj_workflow

```diff
@@ -0,0 +1,284 @@
+#!/bin/env python
+
+# Copyright (C) 2017 Christopher M. Biwer, Alexander Harvey Nitz
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+""" Creates a DAX for a parameter estimation injection study.
+"""
+
+import argparse
+import h5py
+import logging
+import os
+import Pegasus.DAX3 as dax
+import pycbc.workflow as wf
+import pycbc.workflow.inference_followups as inffu
+import pycbc.workflow.minifollowups as mini
+import pycbc.workflow.pegasus_workflow as wdax
+import pycbc.version
+import socket
+import sys
+from pycbc_glue import segments
+from pycbc import results
+from pycbc.results import layout
+from pycbc.types import MultiDetOptionAction
+from pycbc.types import MultiDetOptionAppendAction
+from pycbc.workflow import core
+from pycbc.workflow import plotting
+from pycbc.workflow import WorkflowConfigParser
+
+def to_file(path, ifo=None):
```

Comment:
I think this already exists as a class method:

https://github.com/ligo-cbc/pycbc/blob/master/pycbc/workflow/pegasus_workflow.py#L448